### DISCUSSION TITLE
Tooltip on HxGrid paging component

### DISCUSSION THREAD

**Discussion by sweco-secoma** [havit/Havit.Blazor#414] [Q&A] [answered]
Hi, 

I'm struggling to figure out how to get tooltips on the buttons of the HxPager component for HxGrid. 

I've tried creating a PagerSettings, where i can set up my own button icons - but I can't see a way to get a tooltip. 

Is there a way to do this? - I can't find any examples with tooltips on the pager. 

Thanks!

**Comment by crdo** [comment, marked as answer]
Hi @sweco-secoma, the HxPager is not internally implemented via HxButton and so it does not have the tooltip API that you could use in any straightforward way. There is a possibility to create custom pager wired to the HxGrid but it is not documented yet.

**Reply by sweco-secoma** [reply]
Ok @crdo  thanks for the quick response!
